build.gradle

```diff
@@ -71,7 +71,7 @@ mainClassName = 'com.faforever.moderatorclient.Launcher'
 
 group = 'com.faforever'
 description = 'faf-moderator-client'
-sourceCompatibility = '17'
+sourceCompatibility = '21'
```

Comment:
If the source compatibility is updated then we need to update the github action as well. This may also require updating gradle